content/docs/getting-started/updating-the-ui/index.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Updating the Backstage UI
+lastUpdated: '2021-08-06T12:00:00.0Z'
+description: How to configure the Backstage UI Layout
+---
+
+## Introduction
+
+Roadie allows you to configure which plugins you want to use via our UI. This guide explains the different types of
+plugin components and the different places they can be integrated into the app.
+
+## Prerequisites
+
+* You must be an admin to edit the UI layout. Please complete [this guide](/docs/getting-started/getting-started-for-admins/) before moving on to this one.
+
+## About Plugins
+
+Backstage is a plugin based application where each plugin provides a certain piece of functionality. A plugin may
+integrate with some external system such as GitHub to display information for a certain Entity e.g. Show pull request
+information for a particular service. 
+
+You can read more about Backstage plugins [here](https://backstage.io/docs/plugins/).
+
+In terms of the Backstage layout plugins can provide [three types of component](https://backstage.io/docs/plugins/composability#naming-patterns):
```

Comment:
```suggestion
In terms of the Backstage layout plugins can provide [three types of components](https://backstage.io/docs/plugins/composability#naming-patterns):
```

Reply:
I would try to avoid using the word "component" since it already applies to a formal Backstage concept. How about "format" or "widget"?

```suggestion
Backstage plugins can provide UI elements in [three different formats](https://backstage.io/docs/plugins/composability#naming-patterns):
```

Reply:
@iain-b I think you might have forgotten to push a change here? The text appears unchanged.

Reply:
Nevermind I just amended the commit

Reply:
Yeah I guess. The screenshots are still dark mode on that branch but you said that was fixed in standup.

I'm confused now 😬 .

Reply:
Man this has been a nightmare... 😓  They're there now.